airflow-core/src/airflow/jobs/scheduler_job_runner.py

```diff
@@ -356,6 +356,7 @@ def _executable_task_instances_to_queued(self, max_tis: int, session: Session) -
                 .join(TI.dag_model)
                 .where(~DM.is_paused)
                 .where(TI.state == TaskInstanceState.SCHEDULED)
+                .where(DagModel.bundle_name.is_not(None))
```

Comment:
Let be consistent with elsewhere in the query
```suggestion
                .where(DM.bundle_name.is_not(None))
```

This is so the scheduler won't look at dags that haven't been re-serialized yet for AF3, yeah?